guides/common/modules/proc_configuring-project-with-an-alternate-cname.adoc

```diff
@@ -0,0 +1,18 @@
+[id='configuring-project-with-an-alternate-cname_{context}']
+= Configuring {Project} with an Alternate CNAME
+Use this procedure to configure {Project} with an alternate CNAME. Note that the procedures for users of a default {Project} certificate and custom certificate differ. Note that if you have any hosts registered to {Projec}, you must update the `katello-ca-consumer` package on the hosts after regenerating the {Project} SSL certificate.
+
+.For Default {Project} Certificate Users
+
+* If you have installed {Project} with a default {Project} certificate and want to configure {Project} with an alternate CNAME, enter the following command on {Project} to generate a new default {Project} SSL certificate with an additional CNAME.
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {foreman-installer} --certs-cname _alternate_fqdn_ --certs-reset
+----
+
+* If you have not install {Project}, you can run the `satellite-maintain  --certs-cname _alternate_fqdn_` command  to install {Project} with an alternate CNAME.
```

Comment:
English grammar was never my string suite, but shouldn't that be "If you have not install**ed** ..."?